coverage/lcov.info

```diff
@@ -0,0 +1,28 @@
+TN:
+SF:src/App.tsx
+FN:5,App
+FN:20,(anonymous_1)
+FN:20,(anonymous_2)
+FNF:3
+FNH:0
+FNDA:0,App
+FNDA:0,(anonymous_1)
+FNDA:0,(anonymous_2)
+DA:6,0
+DA:8,0
+DA:20,0
+LF:3
+LH:0
+BRF:0
+BRH:0
+end_of_record
+TN:
+SF:src/main.tsx
+FNF:0
+FNH:0
+DA:6,0
+LF:1
+LH:0
+BRF:0
+BRH:0
+end_of_record
```

Comment:
let's a `coverage/` to .gitignore file

Reply:
removed coverage folder in commit d7b36a5fe217c1452e474749ffc9399785fdc224
added coverage to .gitignore in dfb740eb3fe353cca597f84748fe7c6f14098104